parser-native/src/main/java/ru/urururu/sanity/cpp/NativeBytecodeParser.java

```diff
@@ -40,13 +41,20 @@ private Cfe parseGlobalInitializers(SWIGTYPE_p_LLVMOpaqueModule module) {
         Cfe first = null;
         Cfe last = null;
 
-        SWIGTYPE_p_LLVMOpaqueValue global = bitreader.LLVMGetFirstGlobal(module);
-        while (global != null) {
+        Iterable<SWIGTYPE_p_LLVMOpaqueValue> globals =
+                    Iterables.linked(() -> bitreader.LLVMGetFirstGlobal(module), bitreader::LLVMGetNextGlobal);
+        for (SWIGTYPE_p_LLVMOpaqueValue global : globals) {
             try {
                 SWIGTYPE_p_LLVMOpaqueValue initializer = bitreader.LLVMGetInitializer(global);
                 if (initializer != null) {
                     Cfe cfe;
-                    LValue globalToInitialize = new Indirection(valueParser.parseLValue(null, global));
+                    GlobalVar pointerToGlobal = (GlobalVar) valueParser.parseLValue(null, global);
+
+                    if (pointerToGlobal.getName().contains("rustc_debug")) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS134)